app/models/concerns/cable_ready/updatable.rb

```diff
@@ -93,6 +100,15 @@ def enrich_association_with_updates(name, option)
           options: options
         })
       end
+
+      def broadcast_updates(model_class, options)
+        return if skip_updates_classes.any? { |klass| klass >= self }
```

Comment:
This is a really interesting line. I am happy to admit that I actually do not understand it. Can you break it down for me?

Reply:
Good question! I stole this directly from no_touching[https://github.com/rails/rails/blob/984c3ef2775781d47efa9f541ce570daa2434a80/activerecord/lib/active_record/no_touching.rb#L37](url). 

Essentially, this line makes sure that we do not broadcast if our `skip_updates_classes` include the current class or any class that the current class inherits from. Consider the following structure:

```ruby
class Book < ApplicationRecord
end

class Novel < Book
end
```

If we do the following:

```ruby
Book.skip_updates do
  Novel.first.update!
end
```

the update on `Novel.first` will not be broadcast because `Novel` inherits from `Book` and we decided to skip updates on `Book`.

To be honest, I wouldn't have come up with this if I hadn't read it in the `ActiveRecord` source code, but I do think it makes sense conceptually! 

What do you think?

Reply:
Well, TIL that `any?` supports a block argument, effectively turning it into a `select` that returns a Boolean. Cool!

And I know that you didn't technically come up with it, but I still assign the clever points to you. Thanks for the great explanation. I think that this is ready to go!